spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/SecretsConfigProperties.java

```diff
@@ -155,14 +150,14 @@ public void setLabels(Map<String, String> labels) {
 		}
 
 		public boolean isEmpty() {
-			return StringUtils.isEmpty(this.name) && StringUtils.isEmpty(this.namespace);
+			return !StringUtils.hasLength(this.name) && !StringUtils.hasLength(this.namespace);
 		}
 
 		public SecretsConfigProperties.NormalizedSource normalize(String defaultName, String defaultNamespace,
 				Map<String, String> defaultLabels) {
-			final String normalizedName = StringUtils.isEmpty(this.name) ? defaultName : this.name;
```

Comment:
`isEmpty` is deprecated, replace it with `hasLength`. Also `final` is not really needed